index.php

```diff
@@ -0,0 +1,31 @@
+<?php require('dbconn.php');
+require('dbgetdata.php'); ?>
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Book Club Book Tracker</title>
+    <meta name="author" content="Holly Bedford">
+    <meta name="description" content="Book tracker">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css2?family=Josefin+Slab&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style1.css" type="text/css">
+</head>
+<body>
+<header>
```

Comment:
Watch your indentation in this file! Nested elements get indented inside their parent